fms2_io/fms_netcdf_domain_io.F90

```diff
@@ -26,7 +26,6 @@
 !> @addtogroup fms_netcdf_domain_io_mod
 !> @{
 module fms_netcdf_domain_io_mod
-use, intrinsic :: iso_fortran_env
```

Comment:
These can be removes. They were left over from when [we switch over from the intrinsic types `real32` instead of the portables types from platform.mod](https://github.com/NOAA-GFDL/FMS/issues/674)